boards.txt

```diff
@@ -248,9 +248,18 @@ STCT_nRF52_minidev.menu.softdevice.s132=S132
 STCT_nRF52_minidev.menu.softdevice.s132.softdevice=s132
 STCT_nRF52_minidev.menu.softdevice.s132.softdeviceversion=2.0.1
 STCT_nRF52_minidev.menu.softdevice.s132.upload.maximum_size=409600
-STCT_nRF52_minidev.menu.softdevice.s132.build.extra_flags=-DNRF52 -DS132 -DNRF51_S132
+STCT_nRF52_minidev.menu.softdevice.s132.build.extra_flags=-DNRF52 -DS132 -DNRF52_S132
 STCT_nRF52_minidev.menu.softdevice.s132.build.ldscript=armgcc_s132_nrf52832_xxaa.ld
 
+STCT_nRF52_minidev.menu.board_variant.none=None
+STCT_nRF52_minidev.menu.board_variant.rgz.build.lfclk_flags=-DUSE_LFXO -DSTCT_NRF52_minidev
```

Comment:
Why are the `-DSTCT_NRF52_minidev`, ` -DSTCT_NRF52_minidev_RGZ`, and `-DSTCT_NRF52_minidev_RSM` part of `build.lfclk_flags`?

Reply:
Yeah I made mention of this in https://github.com/sandeepmistry/arduino-nRF5/issues/200#issuecomment-335063303

Its a bit of a hack. It really needs a ```build.variant_flags```, and i didnt know if you wanted to add this just for one use case.

Reply:
I think the `build.variant_flags` is cleaner.